wiki/Active-Effect-Guide.md

```diff
@@ -0,0 +1,442 @@
+
+| Attribute Key | Change Mode | Effect Value |
+| -------- | -------- | -------- |
+| `system.abilities.[abbreviation].value`     | Upgrade     | `[number]`     |
+
+#### Bonus to a Specific Ability Save
+
+| Attribute Key | Change Mode | Effect Value |
+| -------- | -------- | -------- |
+| `system.abilities.[abbreviation].bonuses.save`     | Add     | `+[number/formula]`     |
+
+
+#### Bonus to a Specific Ability Check
+
+| Attribute Key | Change Mode | Effect Value |
+| -------- | -------- | -------- |
+| `system.abilities.[abbreviation].bonuses.check`     | Add     | `+[number/formula]`     |
+
+
+#### Bonus to All Ability Checks
+
+| Attribute Key | Change Mode | Effect Value |
+| -------- | -------- | -------- |
+| `system.bonuses.abilities.skill`     | Add     | `+[number/formula]`     |
+
+
+#### Bonus to All Ability Saves
+
+E.g. Paladin Aura of Protection
+
+| Attribute Key | Change Mode | Effect Value |
+| -------- | -------- | -------- |
+| `system.bonuses.abilities.save`     | Add     | `+[number/formula]`     |
+
+---
+
+### Skills
+```
+system.skills.[abbreviation].bonuses.check
+                                     passive
+```
+
+> <details>
+> <summary>Skill Abbreviations</summary>
+> 
+> | Skill           | Abbreviation |
+> | --------------- | ------------ |
+> | Acrobatics      | `acr`        |
+> | Animal Handling | `ani`        |
+> | Arcana          | `arc`        |
+> | Athletics       | `ath`        |
+> | Deception       | `dec`        |
+> | History         | `his`        |
+> | Insight         | `ins`        |
+> | Investigation   | `inv`        |
+> | Intimidation    | `itm`        |
+> | Medicine        | `med`        |
+> | Nature          | `nat`        |
+> | Persuasion      | `per`        |
+> | Perception      | `prc`        |
+> | Performance     | `prf`        |
+> | Religion        | `rel`        |
+> | Sleight of Hand | `slt`        |
+> | Stealth         | `ste`        |
+> | Survival        | `sur`        |
+> 
+> Source: `CONFIG.DND5E.skills`
+> </details>
+
+
+#### Bonus to a Specific Skill Check
+
+| Attribute Key | Change Mode | Effect Value |
+| -------- | -------- | -------- |
+| `system.skills.[abbreviation].bonuses.check`     | Add     | `+[number/formula]`     |
+
+#### Bonus to a Specific Skill Passive
+
+| Attribute Key | Change Mode | Effect Value |
+| -------- | -------- | -------- |
+| `system.skills.[abbreviation].bonuses.passive`     | Add     | `+[number/formula]`     |
+
+#### Bonus to All Skill Checks
+
+| Attribute Key | Change Mode | Effect Value |
+| -------- | -------- | -------- |
+| `system.bonuses.abilities.skill`     | Add     | `+[number/formula]`     |
+
+#### Bonus to Initiative
+
+Initiative is not quite a skill but behaves similarly.
+
+| Attribute Key | Change Mode | Effect Value |
+| -------- | -------- | -------- |
+| `system.attributes.init.bonus`     | Add     | `+[number/formula]`     |
```

Comment:
We should consider moving this right under 'bonus to a specific ability check' since Initiative is just a special kind of ability check and not really related to skills (aside from skills also being ability checks). We can then avoid the wording about 'not quite a skill'.